### DISCUSSION TITLE
Add new Domain and Tracking Domain via Postal API

### DISCUSSION THREAD

**Discussion by Pispros** [postalserver/postal#2963] [Help with using Postal] [answered]
Hello, is it possible to create a new domain and get a response of SPF, DKIM and Return Path DNS records ?

Same steps to add a tracking domain

**Comment by willpower232** [comment, marked as answer]
That isn't available over the current API I'm afraid but apparently there will be a new one at some point which has more reach inside Postal.

**Reply by Pispros** [reply]
Thank u for your quick response :)
